apmpackage/apm/changelog.yml

```diff
@@ -3,6 +3,11 @@
     - description: Placeholder
       type: enhancement
       link: https://github.com/elastic/apm-server/pull/123
+- version: 8.11.3
```

Comment:
This should point to the latest available release... at the moment it's not correct as I'm simulating this out